### DISCUSSION TITLE
Parametric EQ

### DISCUSSION THREAD

**Discussion by pizzot** [rern/rAudio#515] [Feature Requests]
Is there a plan for adding one? Or some sort of DSP that would compensate for room acoustics? Can provide better listening experience than all the bitperfect stuff :)

**Comment by rern** [comment]
If your equipments and room are decent enough, bit-perfect is the way to go. 
Otherwise an equalizer can be useful not just for the room acoustics but shortcomings in some half decent equipments as well.

This should be ready by next update:

![equalizer](https://user-images.githubusercontent.com/10094236/134855932-7a4e8349-6107-4965-b3fc-09d3abfccac5.jpg)

**Reply by pizzot** [reply]
Thanks for your opinion on my equipment. 

Can't see how a graphic EQ can help people whose equipment and rooms are not decent enough.

**Reply by NexusPenguin** [reply]
Hi,

Is a PEQ not included in Camilla DSP ? 

It hink it is... At least I use PEQ for my headphone in Camilla DSP even if it is quite messy to set-up :-)

Regards.

**Reply by NexusPenguin** [reply]
Hi,

It looks like PEQ is indeed built into Camilla DSP... But if I enable Camilla DSP, whatever the setting, I have an outsample rate of 44.1khz. 

I think it should adapt to the original siamplerate... But as I said : whatever the setting => 44.1Khz
